Source/Core/Core/NetPlayClient.cpp

```diff
@@ -125,7 +125,7 @@ NetPlayClient::NetPlayClient(const std::string& address, const u16 port, NetPlay
 			return;
 		}
 
-		if (!EnsureTraversalClient(centralServer, centralPort))
+		if (!EnsureTraversalClient(centralServer, centralPort, 0))
```

Comment:
Instead of specifying 0 as a magic number here, I think EnsureTraversalClient should have 0 as a default argument for listen_port.